configurations/kmip-ear.yaml

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
@bentsi @roydahan 
FYI
take notice on us adding this new `configuration` folder, for the EaR cases